doc/quint.md

```diff
@@ -256,6 +256,75 @@ Options:
 
    The errors and warnings are written in the format of [ADR002][].
 
+## Command verify
+
+```sh
+$ quint verify <input>
+
+Verify a Quint specification via Apalache
+
+Options:
+  --help             Show help                                     [boolean]
+  --version          Show version number                           [boolean]
+  --how              how to do verification: bounded model checking (bmc) or
+                     symbolic execution (symexec)
+                   [string] [choices: "symexec", "bmc"] [default: "symexec"]
+  --main             name of the main module (by default, computed from
+                     filename)                                      [string]
+  --out              output file (suppresses all console output)    [string]
+  --out-itf          output the trace in the Informal Trace Format to file
+                     (supresses all console output)                 [string]
+  --max-samples      the number of symbolic paths to try (when using
+                     --how=symexec)                [number] [default: 10000]
+  --max-steps        the maximum on the number of steps in every trace
+                                                      [number] [default: 10]
+  --init             name of the initializer action
+                                                  [string] [default: "init"]
+  --step             name of the step action      [string] [default: "step"]
+  --invariant        invariant to check: a definition name or an expression
+                                                [string] [default: ["true"]]
+  --seed             random seed to use for symbolic execution (when using
+                     --how=symexec)                                 [string]
+  --apalache-config  Filename of the additional Apalache flags in the JSON
+                     format                                         [string]
+```
+
+- If there are no critical errors (e.g., in parsing, typechecking, etc.), this
+command sends the Quint specification to the [Apalache][] model checker, which
+will try to find an invariant violation. If it finds one, it prints the trace
+on the standard output. If it does not find a violating trace, it prints the
+longest sample trace that the simulator has found during the execution. When the
```

Comment:
This sentence only applies to one mode. To make it accurate we need a conditional here, which I guess is another indication that theses should be factored into separate commands? :)